sdk/src/main/scala/com/horizen/SidechainNodeViewHolder.scala

```diff
@@ -65,14 +65,14 @@ class SidechainNodeViewHolder(sidechainSettings: SidechainSettings,
 
   override protected def genesisState: (HIS, MS, VL, MP) = {
     val result = for {
-      state <- SidechainState.genesisState(stateStorage, params, applicationState, genesisBlock)
+      state <- SidechainState.createGenesisState(stateStorage, params, applicationState, genesisBlock)
```

Comment:
why methods are renamed?
Previously it was `genesis....` to be exactly in the same way as `nodeviewgholder.genesisState`

Reply:
because we actually create genesis state